spec/client_spec.rb

```diff
@@ -209,7 +209,7 @@
       end
 
       context 'with non-serializable data' do
-        let(:data) { [:foo, 'bar'] }
```

Comment:
Manual validation would be a significant change though, since I imagine we'd want to check nested data as well.

However, why would we want symbols to fail, instead of relying on them being converted to strings?